ddl/reorg.go

```diff
@@ -241,6 +241,14 @@ func (w *worker) runReorgJob(t *meta.Meta, reorgInfo *reorgInfo, tblInfo *model.
 	// wait reorganization job done or timeout
 	select {
 	case err := <-w.reorgCtx.doneCh:
+		if err != nil {
+			w.reorgCtx.clean()
+			return errors.Trace(err)
+		}
+		if w.reorgCtx.isReorgCanceled() {
```

Comment:
why need check reorg canceled here? it will return `dbterror.ErrCancelledDDLJob` if the job is cancled

Reply:
It may return ok even if reorgCancel is set.